src/utils/decimalFormat.ts

```diff
@@ -0,0 +1,15 @@
+import Decimal from "decimal.js";
+import { TokenDetails } from "types";
+
+const SMALLEST_DECIMAL = new Decimal(0.00001);
+
+export const decimalFormat = (
+  decimal: Decimal,
+  token: TokenDetails
+): string => {
+  if (decimal.lt(SMALLEST_DECIMAL)) {
+    return `<${SMALLEST_DECIMAL.toString()} ${token.symbol}`;
+  }
+
+  return decimal.toSD(4).toString();
```

Comment:
0.00001 will return 0.0000 while 0.000009 will return <0.00001, no?

Besides that, I'll eventually finish this https://github.com/gnosis/dex-js/pull/204 and consolidate how we do the formatting.
In my mental todo list.

Reply:
Good catch! WIll fix before moving this